### PR TITLE
feat: pending files banner

### PR DESCRIPTION
**Pull request by anri-asaturov**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/9350/in-app-notice-regarding-downloading-pending-files

#### Testing instructions  
- make sure new account is not shown the banner
- make sure existing account are shown banners when have pending files
- make sure banner is not coming back with restart when dismissed

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ebilg** [comment]
can be merged

File: src/ui/files/components/PendingFilesBanner.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  width: 100%;
+  min-height: $pending-files-banner-height;
+  background-color: $peerio-purple;
+  padding: 0 16px;
+
+  .banner-icon {
+    flex: 0 0;
+    font-size: 20px;
+    border: 4px solid rgba($peerio-purple, 0.87);
+    background-color: white;
```
Comment (suprko) [inline src/ui/files/components/PendingFilesBanner.scss]: use `$white` sass variable

File: src/ui/files/components/PendingFilesBanner.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.pending-files-banner {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  width: 100%;
+  min-height: $pending-files-banner-height;
+  background-color: $peerio-purple;
+  padding: 0 16px;
```
Comment (suprko) [inline src/ui/files/components/PendingFilesBanner.scss]: we have padding/margin variables, unfortunately they are tucked away in PeerUI :weary: anyway 16px is i think `$padding-default`

File: src/ui/files/components/PendingFilesBanner.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  .banner-icon {
+    flex: 0 0;
+    font-size: 20px;
+    border: 4px solid rgba($peerio-purple, 0.87);
+    background-color: white;
+    border-radius: 30px;
+    min-width: 40px;
+    height: 40px;
+    text-align: center;
+    margin-right: 24px;
```
Comment (suprko) [inline src/ui/files/components/PendingFilesBanner.scss]: similarly, this is `$margin-large`

File: src/ui/files/components/PendingFilesBanner.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    .material-icons {
```
Comment (suprko) [inline src/ui/files/components/PendingFilesBanner.scss]: just a code style thing: we've been putting a extra line break before new selectors (so here, put an extra line break above `.material-icons` to clearly separate it from `margin-right`)

File: src/ui/files/components/PendingFilesBanner.scss (hunk 1 above)
Comment (anri-asaturov) [reply src/ui/files/components/PendingFilesBanner.scss]: why do we need $white variable? can white color ever change? :)

File: src/ui/files/components/PendingFilesBanner.scss (hunk 4 above)
Comment (anri-asaturov) [reply src/ui/files/components/PendingFilesBanner.scss]: @lostfictions can we have prettier deal with it?

Comment (anri-asaturov) [reply src/ui/files/components/PendingFilesBanner.scss]: Ok i've just added stylelint and stylelint --fix in the pipeline.

**Pull request merged**